src/pages/ErrorPage.css

```diff
@@ -0,0 +1,22 @@
+h1 {
```

Comment:
if we're modifying just one `h1` here let's just give the `h1` on the error page a `className` and add the margin block property to that class selector instead of to h1 as a whole